wellcomeml/ml/bert_semantic_equivalence.py

```diff
@@ -54,6 +56,7 @@ def __init__(
             random_seed: A seed used for shuffling the dataset upon training
             buffer_size: A buffer size that will be used when shuffling the dataset
         """
+        self.logger = build_logger(logging_level, __name__)
```

Comment:
potentially better to initialise logger outside the init

Reply:
I'm not sure where else I'd initialise it...

Reply:
I was thinking start of the file